src/Shared/TaskToApm.cs

```diff
@@ -0,0 +1,121 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
This is a different license than our other dependencies. You may need to change it or add an exclusion?

Reply:
@Pilchie I know you're on vacation but any ideas?
@richlander Any ideas what to do here?